.circleci/config.yml

```diff
@@ -133,33 +110,86 @@ jobs:
           upload_name: Auth0 Flutter Android
           flags: auth0_flutter_android
   smoke_test_android:
-    parameters:
-      android_simulator:
-        type: string
-    executor:
-      name: android/android-machine
-      resource-class: xlarge
-      tag: 2022.06.1
+    machine:
+      image: android:2022.12.1
+      # To optimize build times, we recommend "large" and above for Android-related jobs
+      resource_class: xlarge
     steps:
-      - run:
-          name: Guard Smoke Tests
-          command: |
-            if ! [[ -z "$CIRCLE_PR_NUMBER" ]]; then
-                circleci-agent step halt
-            fi
```

Comment:
One last thing, we should probably keep this to avoid running integration tests on PRs from forks.